scripts/visualization/streamlit_mapa_licea.py

```
@@ -427,6 +441,23 @@ def load_quality_cached(
 
+@st.cache_data(show_spinner=False)
+def load_year_sheet_cached(
+    excel_file: Path,
+    sheet_name: str,
+    selected_year: int | None,
+    data_version: int,
+) -> pd.DataFrame:
+    _ = data_version
+    try:
+        df = pd.read_excel(excel_file, sheet_name=sheet_name)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)